### ISSUE TITLE
Q: project structure about msbuildproj file

### ISSUE THREAD

**Issue by liesauer** [peachpiecompiler/peachpie#735]
Does a peachpie project must use `msbuildproj`? can we change to `csproj` with full compatibility? it looks like `msbuildproj` is not supported everywhere. like GitHub, `msbuildproj` can not be analyzed. see [Supported package ecosystems](https://help.github.com/en/github/visualizing-repository-data-with-graphs/listing-the-packages-that-a-repository-depends-on#supported-package-ecosystems).

**Comment by jakubmisek** [comment]
`.csproj` is recognized by toolings, IDEs and common msbuild tasks as a C# project - this might cause you some compatibility troubles. In most cases, however, it is just fine (just not nice). (We used the same trick to workaround some JetBrains Rider restrictions tho)

The only requirement is, the project file extension must be a known project extension (`.proj`, `.msbuildproj`, ...) so it can be opened by Visual Studio, and it should be suffixed with `***proj` so `dotnet` recognizes it as a project file.

I'm not sure if it's worth using confusing file extension just be listed on GitHub packages :) anyways the recommendations above are all I have.

tldr; you can rename your project file to anything, mostly it will work fine.
